README.Rmd

```diff
@@ -60,8 +58,8 @@ To be included as an `{admiral}` package extension we expect developers to adher
 
 * `r dyn_link("Code of Conduct", admiral_homepage, "CODE_OF_CONDUCT.html")`
 * `r dyn_link("Manifesto", admiral_homepage, "index.html#admiral-manifesto")`
-* `r dyn_link("Programming strategy", admiral_homepage, "articles/programming_strategy.html")`
-* `r dyn_link("Development process", admiral_homepage, "articles/development_process.html")`
+* [Programming strategy](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy)
+* [Development process](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html)
```

Comment:
Should we link to the devel version?

Reply:
good question! @thomas-neitmann @bms63 @cicdguy didnt we say for admiraldev we'd have the website being maintained from the latest `devel` version of the package as default? e.g. maybe if i deleted the `main` above would that automatically push user to `devel` anyway

Reply:
Not sure.  We were just trying to update this to the `devel` and looks like the site is not working at the moment

Reply:
whatever gets decided here could be a new issue though as would need same updating across all admiral packages. i'm going to merge this PR for now